databuilder/extractor/vertica_metadata_extractor.py

```diff
@@ -0,0 +1,140 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from collections import namedtuple
+from itertools import groupby
+from typing import (
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree
+
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.sql_alchemy_extractor import SQLAlchemyExtractor
+from databuilder.models.table_metadata import ColumnMetadata, TableMetadata
+
+TableKey = namedtuple('TableKey', ['schema', 'table_name'])
+
+LOGGER = logging.getLogger(__name__)
+
+
+class VerticaMetadataExtractor(Extractor):
+    """
+    Extracts vertica table and column metadata from underlying meta store database using SQLAlchemyExtractor
+    V_CATALOG does not have table and column description columns
+    CLUSTER_KEY config parameter is used as cluster name
+    Not distinguishing between table & view for now
+    """
+    # SELECT statement from vertica information_schema to extract table and column metadata
```

Comment:
do you need any vertica deps to use this extractor?

Reply:
Sure, I can add a section in the README in the top directory of the repo, under the List of Extractors.

I can also add an example as a file example/scripts/vertica_data_loader.py that shows how the extractor can be used.